src/CalcViewModel/DateCalculatorViewModel.cpp

```diff
@@ -159,14 +172,21 @@ void DateCalculatorViewModel::UpdateDisplayResult()
 {
     if (m_IsDateDiffMode)
     {
-        // Are to and from dates the same
-        if (m_dateDiffResultInDays.day == 0)
+        if (m_dateDiffResultInDays == DateDifferenceUnknown)
+        {
+            IsDiffInDays = false;
+            StrDateDiffResultInDays = L"";
+            StrDateDiffResult = L"";
```

Comment:
>L"" [](start = 32, length = 3)

I wasn't able to hit this case when playing around, but it's probably worth adding a "Date calculation failed" message.  A blank result can give the user the impression that something is going on when in fact, nothing is still running.

Reply:
This case _should_ never happen but we still need to manage it (in case of a future diff breaks it).

I will change the string.